cleo_plugins/Math/Math.cpp

```diff
@@ -434,4 +446,96 @@ class Math
         OPCODE_CONDITION_RESULT(value != 0);
         return OR_CONTINUE;
     }
-} Math;
+
+    //2705=-1,pick_random_int values %d% store_to %d%
```

Comment:
picking random array element is called sampling

please rename these it `sample_int`, `sample_float`, `sample_string`

Reply:
Hm not sure about that. I would never guess it takes random element from set.
"Sample - get a representative experience of". Especially in software development it is already used for totally different meanings (like texture sampling).
Originally I considered also "select". Does some programming language have similar function?

Reply:
pick (noun) · picks (plural noun)
    an act or the right of selecting something from a number of alternatives

I was baffling about name already and this seem to be the best one. Other options seems like fancy ways to not say "pick".

**random_pick_int** maybe?

Reply:
sample is a verb too

Reply:
https://www.geeksforgeeks.org/python-random-sample-function/
https://lodash.com/docs/4.17.15#sample

Reply:
Yeah, like sampling input sensor by averaging input over sampling window time, or sampling audio to fill given time by repeating small portion. 
I think all randomization opcodes should have word "random" in name. Then searching in SBL actually provides user with options to select from.
**Math.Sample(1,2.3)** looks meaningless.